src/content/docs/features/chwd.md

```diff
@@ -0,0 +1,92 @@
+---
+title: CachyOS Hardware Detection
+description: Hardware Detection and Configuration for CachyOS
+---
+
+[CachyOS Hardware Detection](https://github.com/CachyOS/chwd/) or better known as **`chwd`** is our hardware detection tool that
+enables us to power a variety of hardware.
+This includes systems running NVIDIA's graphics cards, T2 Macbooks and handheld devices such as Steam Deck and ROG Ally.
+
+## Usage
+
+**`chwd`** is typically ran during installation time to provide the necessary packages for your system. However, it is also possible
+to use it post-install.
+
+### Auto Configuration via Class IDs
+
+We can get class IDs by running `lspci`
+
+```sh
+❯ lscpi -nn
```

Comment:
Instead of using lspci, you can get a list of available profiles via: ``chwd --detail --list``.